src/internal/modules/invoice-accounts/controllers/invoice-accounts.js

```diff
@@ -53,35 +55,74 @@ const postCompany = async (request, h) => {
 
 const getAddress = async (request, h) => {
   const { regionId, companyId } = request.params;
-  const addresses = await dataService.getCompanyAddresses(companyId);
   // get the session data to check if the address has been set and if it is new or existing
   const session = dataService.sessionManager(request, regionId, companyId);
+  const addresses = await helpers.getAllAddresses(companyId, session);
   // @TODO this might need a mapper to map the session address data to the Company address shape passed to the form
```

Comment:
can we remove this comment now?

Reply:
I don't know - I didn't put it there 😂 

I'll remove it now anyway.